config/environments/development.rb

```diff
@@ -87,7 +87,7 @@
   config.action_mailer.smtp_settings = {
     address: 'smtp.gmail.com',
     port: 587,
-    user_name: 'eventnxtapp@gmail.com', # to whatever email you want to send email from
+    user_name: 'suryacherukuri999@gmail.com', # to whatever email you want to send email from
```

Comment:
Again. Not sure about this. I think it's better we send email from the event orignaizer email. That is, get the email from the user model, but I don't know if it is possible or not.